README.md

```diff
@@ -20,4 +20,19 @@
     `node server.js`
 
 - Hit [http://localhost:3000](http://localhost:3000) on your favorite browser.
-- Learn and hack it!
\ No newline at end of file
+- Learn and hack it!
+
+## Usage With Docker
+- Make sure you have installed docker and docker-compose on your machine
+- Clone/download this repo.
+- Enter the directory.
+
+  `cd belajar-node`
+
+- Run the app with docker-compose
+
+    `sudo docker-compose up`
+
+- Hit [http://localhost:3000](http://localhost:3000) on your favorite browser to open the apps.
+- use [http://localhost:4000](http://localhost:4000) to open adminer and manage database.
+- use PORT `33060` to connect mysql database from your mysql client.
```

Comment:
Change it to `Use PORT `33060` to connect to MySQL database from your MySQL client.`